### ISSUE TITLE
Contents edit page (article, question 공통 / markdown editor 제외)

### ISSUE THREAD

**Issue by HuanSuh** [flutterstudygn/stackoverflutter#22]
(page_contents_edit)

**Comment by Deockyoung** [comment]
![edit](https://user-images.githubusercontent.com/24905164/71638941-81b1c980-2cb0-11ea-8472-c32d3b668f08.PNG)

**Comment by Deockyoung** [comment]
Title, Comments, Tags 부분 만들면 될까요???

**Comment by Sugyo-In** [comment]
@Deockyoung 
제가 보기엔 edit 페이지가 아니라 게시글 페이지를 작성해야할 것 같은데요.

**Comment by Deockyoung** [comment]
@Sugyo-In 게시글 페이지는 어디인가요?
![image](https://user-images.githubusercontent.com/24905164/71650842-479e0180-2d5c-11ea-935a-03008ca3b1d3.png)
여기를 말씀하시는건가요??

**Comment by Sugyo-In** [comment]
@Deockyoung 
아 죄송해요. 제가 글을 잘 못 봤네요;
덕영님이 생각하신 부분이 맞는 것 같아요.

**Comment by Deockyoung** [comment]
@Sugyo-In 아 그렇군요 ㅋㅋㅋ다행이네요

**Comment by HuanSuh** [comment]
네, edit 페이지는 여기가 맞습니다! Contents 부분에 markdown editor도 외부 라이브러리 참고해서 별도 컴포넌트 작업해두실 수 있으면 그렇게 진행 부탁드릴게요

**Comment by Deockyoung** [comment]
@HuanSuh 죄송하지만 여기라 함은 

![image](https://user-images.githubusercontent.com/24905164/71651124-87fe7f00-2d5e-11ea-9b8f-055242589a42.png)


여기를 말씀하시는건가요?

**Comment by HuanSuh** [comment]
네네

**Comment by Deockyoung** [comment]
@HuanSuh 네 알겠습니다~~!

**Comment by Deockyoung** [comment]
![image](https://user-images.githubusercontent.com/24905164/71667186-4fcf5e80-2da7-11ea-88f8-4e30a8caad12.png)

요것을 넣어야 하는데....
zefyr 0.8.0 패키지가 있긴 하더라구요 웹에서도 적용될진 모르겠는데..혹시 추천해주실만한거 있나요???

**Comment by Deockyoung** [comment]
tags 
flutter_tags 0.4.5
사용

**Comment by Deockyoung** [comment]
flutter_tagging 2.2.0+1
https://pub.dev/packages/flutter_tagging

**Comment by Deockyoung** [comment]
> ![image](https://user-images.githubusercontent.com/24905164/71667186-4fcf5e80-2da7-11ea-88f8-4e30a8caad12.png)
> 
> 요것을 넣어야 하는데....
> zefyr 0.8.0 패키지가 있긴 하더라구요 웹에서도 적용될진 모르겠는데..혹시 추천해주실만한거 있나요???

https://pub.dev/packages/angular_quill
https://pub.dev/packages/yabwe#-readme-tab-

**Comment by HuanSuh** [comment]
엇 꼭 저대로 하신다기보단 마크다운 입력만 할 수 있는 폼이면 될거같아요

**Comment by Deockyoung** [comment]
@HuanSuh 아 저부분이 마크다운에디터였군요... 다른걸로 착각하고있었어요 ㅋㅋㅋ
